packages/components/psammead-story-promo/src/index.stories.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { Fragment } from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import { inputProvider } from '@bbc/psammead-storybook-helpers';
+import Img from '@bbc/psammead-image';
+import { latin } from '@bbc/gel-foundations/scripts';
+import Timestamp from '@bbc/psammead-timestamp';
+import notes from '../README.md';
+import StoryPromo, { Headline, Summary } from './index';
+
+const ImageComponent = (
+  <Img
+    alt={text('Image alt text', '2019-03-01T14:00+00:00')}
+    src={text(
+      'Image src',
+      'https://ichef.bbci.co.uk/news/660/cpsprodpb/909B/production/_106491073_gettyimages-909260196.jpg',
+    )}
+    width="100%"
```

Comment:
Based on the [docs](https://github.com/bbc/psammead/blob/latest/packages/components/psammead-image/README.md) this is meant to be the `px` value of the image not the css being applied

Reply:
Fixed :)